backend/typescript/middlewares/givecloud.ts

```diff
@@ -0,0 +1,20 @@
+import { Request, Response, NextFunction } from "express";
+
+/* 
+checks the .env file to see if the givecloud webhook is enabled, 
```

Comment:
(nit) This checks everywhere environment variables are set, not just the `.env` file. (ie we could set the variable on Heroku too when deployed)